libs/create-zephyr-apps/src/templates.ts

```diff
@@ -19,65 +12,122 @@ export const ProjectTypes = [
 ];
 
 // TODO: Programmatically load templates from the examples repo after cloning it
-export const Templates: { name: string; label: string; hint: string }[] = [
+export const Templates = [
```

Comment:
we can define a type for that:
type Template = {
  name: string;
  label: string;
  hint: string;
  directory: string;
};
and set

```
export const Templates: Template[] = [
```

Reply:
Added